mvt/layer_test.go

```diff
@@ -1,10 +1,12 @@
 package mvt
 
 import (
+	"reflect"
 	"testing"
 
 	"context"
 
+	"github.com/gdey/tbltest"
```

Comment:
What's the benefit of using this over simply putting the function code in a loop that cycles over the testcases at the bottom of the test?  If @ARolek gives the thumbs-down on a well-reviewed and stable package like testify just for the effort of having an additional package to justify, what's the justification for using code from personal repositories?